### PR TITLE
Topic recognition

### PR DESCRIPTION
**Pull request by Pragatheeswari-dev**
Performing this pull request, as part of "Report - Pattern Recognition". I have done Task-6 DCGAN model on OASIS Dataset. In addition, I would  like to bring to your notice that assessment was commenced only on 1st November due to medical issues, evidence can be provided if needed.

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
- Sufficiently detailed commit messages
- Code valid for Hard Difficulty with minor technical mistakes
- Good results

Other comments
- task 6 in a folder called task 4 is a bit confusing 
- indentation problem in notebook, may note have mattered
- Please follow python file  and folder naming convention e.g. `lorem_ipsum_123_abc.py`
- Please group the `.png` files and put them in a sub-directory
- `elu` is not a valid output activation function. You may want to include a reference if its proven to have worked better.

**Comment by rexsw** [comment]
Code looks good for hard difficulty

For next time;

Comments such as 
     # compile model
     model.compile(loss='binary_crossentropy',
Add nothing to the readability of the code and should be avoided. Overall comments were well done

Code in files like driver scripts should be in a main function and in a python file.

**Comment by Pragatheeswari-dev** [comment]
Made updates based on the tutor feedback - modified folder structure. 
And added header block in the gan_model.py as well.

**Comment by shakes76** [comment]
## TF Usage
OK
## Good Practice (Design/Commenting)
Good design and comments
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage, Readme OK
Module present
## Commit Log
Meaningful commit messages 
Progressive commits used 
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
**Feedback required. move image files into sub directory.**
No Request Description. In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected. Remember you could use/reuse the README info. 
Notebooks have markdown sections for you to explain what's going on, so you should make use of it.

**Comment by Pragatheeswari-dev** [comment]
Hi Shakes, 
             Thanks for your feedback. I have created a sub-directory called "Images" and moved all my image files in there. Made corresponding updates for image-URL in the README.md file as well.  Let me know if I need to make any further updates.

Cheers.
